### PR TITLE
check in of master and docinfo for books

### PR DESCRIPTION
**Pull request by vikram-redhat**

### CODE REVIEW

**Comment by bcpswope** [comment]
Hi. I need the access console topic, very important topic....  looks like it was removed here but I cannot see a reason in my nav file bc it is working.

Please hold off on this merge until I get clarity about this file.

File: manage_cluster/docinfo.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>Manage Cluster</subtitle>
+<abstract>
```
Comment (bcpswope) [inline manage_cluster/docinfo.xml:4]: I don't want to mess you up by changing myself, but can we please lower-case the "cluster" in line 3? Thanks!

Comment (vikram-redhat) [reply manage_cluster/docinfo.xml:4]: Done.

File: security/docinfo.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>Security</subtitle>
+<abstract>
+    <para>Security and Governance in {product-title}</para>
+</abstract>
```
Comment (bcpswope) [inline security/docinfo.xml:6]: same here, can we go with a lowercase 'g'

Thanks!

File: manage_cluster/docinfo.xml (hunk 1 above)
Comment (bcpswope) [reply manage_cluster/docinfo.xml:4]: oh wow, that was fast!

**Comment by vikram-redhat** [comment]
> Hi. I need the access console topic, very important topic.... looks like it was removed here but I cannot see a reason in my nav file bc it is working.

It's removing the one with the extra space in the name. I thought I had removed it in an earlier commit, but it seems to have snuck back.

**Comment by bcpswope** [comment]
Ok, I see. Thanks @vikram-redhat are you ready for a merge!?

**Pull request merged**